addons/strings.yaml

```
@@ -112,3 +112,26 @@ tryFirefoxExtension:
   downloadFirefox:
     value: Download Firefox
     comment: Button label
+
+upgradeToBundle:
+  upgrade:
+    value: Upgrade
+    comment: Text for upgrade button
+  title:
+    value: 3 tools, 1 privacy plan — VPN, email masking, and data protection
+    comment: Title for message
+  line1:
+    value: For $8.25/month, get our privacy tools in one plan.
+    comment: First line of message. This message is only being shown to users who purchase VPN in USD currency, so please use something explicit like USD$8.25.
```

```suggestion
    comment: First line of message. This message is only being shown to users who purchase VPN in USD currency, so please keep the amount and currency unchanged (USD$8.25).
```

Do we support variables? I'm really concerned about this being translated by mistake, especially with pretranslation, so I'd be a lot more comfortable if there was no space for errors.

---

~We don't support random variables~. Thinking about it, we support a single variable to swap in per a message. This was intended for versions for the update and what's new messages (and is named `shortVersion` as a reflection of that), but I just tested and we can just as easily use this variable for the currency. 

(Notably, our current system would *not* permit adding in a per-locale price - every single locale would get `$8.25` without a bunch of work to improve this system - but this should work for the current use case.)

New code is now up.